card/lib/card/active_record_helper.rb

```diff
@@ -5,8 +5,9 @@ def create_card args
       Card.create! args
     end
 
+    # if card with same name exists move it out of the way
     def create_card! args
-      create_card args.reverse_merge(rename_if_conflict: :new)
+      create_card args.reverse_merge(rename_if_conflict: :old)
```

Comment:
I think :new is probably a better default, because it puts the burden on the new migration to use codenames.